recipes/dist-upgrade.md

```diff
@@ -8,8 +8,24 @@ Upgrading to the latest greatest (and experimental) releases is quite simple:
 2. Perform dist-upgrade:
 
     ```bash
-    apt-get update
-    apt-get dist-upgrade
+    sudo apt-get update
+    sudo apt-get dist-upgrade
     ```
 
-3. Reboot
+3. If you are running a desktop environment or X11 setup, see the below note on X11.
+
+4. Reboot
+
+## X11 and older releases
+
+When upgrading from older releases, some users have reported issues with X11 no longer starting. If you are upgrading an older (i.e. `0.5.x`) release with a desktop environment, it is advisable that you also do a 
```

Comment:
put the `before rebooting the system:` here.

Reply:
Like that?